projects/storefrontlib/src/lib/my-account/order/order-details/order-details.component.ts

```diff
@@ -21,7 +18,7 @@ export class OrderDetailsComponent implements OnInit, OnDestroy {
     private routingService: RoutingService
   ) {}
 
-  order$: Observable<any>;
+  order$: Observable<Order>;
   subscription: Subscription;
 
   ngOnInit() {
```

Comment:
Can you add a type:
* on line 92 - `getConsignmentProducts()`
* on line 93 - `const products = [];`